src/Language/TypeScript/Assignment.hs

```diff
@@ -34,7 +34,7 @@ import qualified Language.TypeScript.Syntax as TypeScript.Syntax
 import qualified Language.TypeScript.Resolution as TypeScript.Resolution
 import Language.TypeScript.Term as TypeScript
 import Prologue
-import TreeSitter.TypeScript as Grammar
+import Language.TypeScript.Grammar as Grammar
```

Comment:
I got `Language.TypeScript.Grammar` compiling fine by adjusting the file path to be one level less nested than the rest, but I get this error when I try to load `Assignment` or `Language.TypeScript` 

```
[71 of 71] Compiling Language.TypeScript.Assignment ( /Users/aymannadeem/github/semantic/src/Language/TypeScript/Assignment.hs, dist-newstyle/build/x86_64-osx/ghc-8.8.1/build-repl/Language/TypeScript/Assignment.o )

/Users/aymannadeem/github/semantic/src/Language/TypeScript/Assignment.hs:37:1: error:
    Could not find module ‘Language.TypeScript.Grammar’
    Perhaps you meant
      Language.TypeScript.Syntax
      Language.TypeScript.Tags
      Language.TypeScript.Term
    Use -v (or `:set -v` in ghci) to see a list of the files searched for.
   |
37 | import Language.TypeScript.Grammar as Grammar
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Failed, 70 modules loaded.
```